tools/cxbe/Error.cpp

```diff
@@ -54,9 +55,9 @@ bool Error::ClearError()
 void Error::SetError(const char *x_szError, bool x_bFatal)
 {
     if(m_szError == 0)
-        m_szError = new char[256];
+        m_szError = new char[ERROR_LEN+1];
```

Comment:
nit: I just realized that I should initialize `m_szError[ERROR_LEN] = '\0'`; otherwise a long `x_szError` will not be zero-terminated in `strncpy` below (as it's not initialized by `new char[..]`).

Other instances of such `strncpy` might exist.
I don't consider this a blocking issue as the problem had already existed anyway.

Reply:
This should be fixed now.